.gitignore

```diff
@@ -200,7 +200,7 @@ PublishScripts/
 # NuGet Symbol Packages
 *.snupkg
 # The packages folder can be ignored because of Package Restore
-**/[Pp]ackages/*
+*/**/[Pp]ackages/*
```

Comment:
Is it required to call the folder packages? Maybe, instead of changing the rule, we change the folder name?
BTW, if they are treated as source code, do you want to put them under `src` folder?

Reply:
I don't think we should put them under src, these are really more in the realm of distributions as it stands. If/When we have more complex projects we can take a look at that but what's currently under the packages repo is really just generated output and definitions.